source/fundamentals/crud/read-operations/geo.txt

```diff
@@ -0,0 +1,306 @@
+===================
+Search Geospatially
+===================
+
+.. default-domain:: mongodb
+
+.. contents:: On this page
+   :local:
+   :backlinks: none
+   :depth: 2
+   :class: singlecol
+
+Overview
+--------
+
+On this page, you can learn how to query **geospatial data**
```

Comment:
Thanks for the note. Committing suggested change. 

This sentence was rewritten in Team PR review and I think Chris has a  preference for the "you can learn" phrasing. I personally don't have a preference between the phrasings, but I think we should reach consensus as a team on which phrasings are acceptable (as I understand it you prefer the "you will" phrasing and Chris prefers the "you can" phrasing in introductions). As it stands now, *no matter which one a writer picks it will be requested to be changed by someone on the team* (which seems incorrect). 

I am committing the suggested change, but I think we should reach consensus on this issue as a team and maybe make more rigid style guidelines for our introductions such that we have a clear definition of success for writers.